system/os_host/build_ds203.sh

```diff
@@ -19,7 +19,7 @@ INCLUDES="\
   -I ../library/CMSIS/Device/STM32F10x/Include \
   -I ../library"
 
-arm-none-eabi-gcc -mcpu=cortex-m3 -mthumb -c ../source/bios/ds203/BIOS.s -o BIOS.o
+arm-none-eabi-gcc -mcpu=cortex-m3 -mthumb -c ../source/bios/ds203/BIOS.S -o BIOS.o || exit 1
```

Comment:
Stop build if there's an error. Currently the scripts try to continue building other stuff although something crashes.